packages/components/src/PassengerCard/PassengerCard.js

```diff
@@ -0,0 +1,135 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BagInformation from './BagInformation';
+import Separator from '../separator/Separator';
+
+type Bag = {|
+  +count: number,
+  +type: string,
+|};
+
+type Props = {|
+  +blank: boolean,
+  +name: string,
+  +gender: 'female' | 'male' | 'other',
+  +nationality: string,
+  +dateOfBirth: string,
+  +id: string,
+  +insurance: string,
+  +bags: Array<Bag>,
+  +passengerCount: number,
+|};
+
+export default function PassengerCard(props: Props) {
+  const newPassengerTitle = `${props.passengerCount}. Passenger`;
+  const preffix =
+    props.gender === 'male' ? 'Mr.' : props.gender === 'female' ? 'Ms.' : '';
+  const passengerTitle = `${preffix} ${props.name}`;
```

Comment:
Actually title in english is Mr. or Ms. So this could be something like `passengerWithTitle` and `prefix` could be just `title`, what do you think?

Reply:
@tbergq We need to get some Translation setup going, WDYT? Otherwise, we will need a huge refactor at some point.

Reply:
No, this is an example app for developers, they should be able to understand english

Reply:
> Actually title in english is Mr. or Ms. So this could be something like `passengerWithTitle` and `prefix` could be just `title`, what do you think?

Yes it would be better

Reply:
@tbergq  But still, we could showcase how we do translation?

Reply:
> we could showcase how we do translation?

IMHO it is not necessary at this point.